prinz/src/test/scala/pl/touk/nussknacker/prinz/UnitIntegrationTest.scala

```diff
@@ -23,7 +24,9 @@ object UnitIntegrationTest {
 
   private val DOCKER_COMPOSE_FILE = new File("../dev-environment/docker-compose.yaml")
 
-  private val MODEL_SERVED_READY_REGEX = s".*Listening at.*1234.*"
+  private val MODEL_1_SERVED_READY_REGEX = s".*Listening at.*1234.*"
+
+  private val MODEL_2_SERVED_READY_REGEX = s".*Listening at.*4321.*"
```

Comment:
Can we take port number from somewhere?

Reply:
we can assume it is defined in env variables and the n the tests in scala as well as docker env would take the same value, to be done in future if needed